examples/getstarted/package.json

```diff
@@ -12,7 +12,7 @@
     "strapi": "strapi"
   },
   "dependencies": {
-    "@strapi/icons": "1.7.9",
+    "@strapi/icons": "0.0.0-3cf3ff39",
```

Comment:
Is this version correct? Should it not be 1.8.0 ?

Reply:
No the DS is currently released as an experimental release and the PR is still open.

Reply:
For reference https://github.com/strapi/design-system/pull/985